### PR TITLE
[ticket/14202]Add missing sr-only classes

### PR DESCRIPTION
**Pull request by hanakin**
ticket: https://tracker.phpbb.com/browse/PHPBB3-14202
PHPBB3-14202

### CODE REVIEW

**Comment by hanakin** [comment]
!set Styles

File: phpBB/styles/prosilver/template/forum_fn.js [hunk 1]
```diff
@@ -579,8 +579,10 @@ function parseDocument($container) {
 
 				if ($this.hasClass('post-buttons')) {
 					$('.button', $menuContents).removeClass('button');
-					$('.js-responsive-menu-link').addClass('button');
+					$('.sr-only', $menuContents).removeClass('sr-only');
+					$('.js-responsive-menu-link').addClass('button').addClass('button-icon-only');
 					$('.js-responsive-menu-link .icon').removeClass('fa-bars').addClass('fa-ellipsis-h');
+
```
Comment (bantu) [inline phpBB/styles/prosilver/template/forum_fn.js]: I don't believe we want this empty line.

**Comment by hanakin** [comment]
ready for review

**Comment by hanakin** [comment]
@cyberalien @VSEphpbb

**Comment by cyberalien** [comment]
Looks fine

**Pull request merged**